addons/test_timer/tests/test_timer.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import TransactionCase
```

Comment:
Could you convert you test into a SavepointCase. That way we avoid one test to influence another. ;)